python/ray/serve/_private/http_state.py

```diff
@@ -238,14 +320,39 @@ def _get_target_nodes(self) -> List[Tuple[str, str]]:
 
         return target_nodes
 
+    def _generate_actor_name(self, node_id: str) -> str:
+        return format_actor_name(SERVE_PROXY_NAME, self._controller_name, node_id)
+
+    def _create_new_proxy(
+        self, name: str, node_id: str, node_ip_address: str
+    ) -> ActorHandle:
```

Comment:
nit: to match other naming (e.g., `_start_proxies_if_needed`)
```suggestion
    def _start_proxy(
        self, name: str, node_id: str, node_ip_address: str
    ) -> ActorHandle:
```